Backend/src/main/java/idealab/api/model/Queue.java

```diff
@@ -25,6 +25,8 @@
     @Column(name = "rank", nullable = false)
     private Integer rank;
 
+    public Queue() {}
```

Comment:
I know you didn't write this class but any chance you could change it's name to what the queue is for? Ex print queue? It is confusing (to me) to have it just called Queue since there is already a Java interface [called Queue](https://docs.oracle.com/javase/7/docs/api/java/util/Queue.html). Lmk if you disagree.

Reply:
Done!